server/src/controllers/onNewTask.ts

```diff
@@ -0,0 +1,20 @@
+import * as socketio from 'socket.io';
+import { rooms } from 'models/Rooms';
+
+interface NewTask {
+  task: string;
+  roomId: number;
+}
+
+const onNewTask = (io: socketio.Server, socket: socketio.Socket) => ({ roomId, task }: NewTask) => {
+  const room = rooms.getRoom(roomId);
+  room.task = task;
+
+  const message = task;
+
+  socket.join(roomId.toString());
```

Comment:
Dlaczego tu mamy socket.join? Jeżeli dany client może wpisać task to siłą rzeczy musi już być w pokoju